### ISSUE TITLE
Add button to open website like wikipedia field

### ISSUE THREAD

**Issue by boothym** [openstreetmap/iD#3312]
The wikipedia field has a button which opens the page listed in the text box - would be nice if it was added to the website field as well.
![url](https://cloud.githubusercontent.com/assets/16965772/17310744/40b36bd6-583e-11e6-8375-93340f0077f0.png)

**Comment by bhousel** [comment]
See #3075
>I don't think it's a good idea to link to untrusted resources. A malicious mapper could put anything in the website= tag, and trick unsuspecting users into visiting that link.

**Comment by homersimpsons** [comment]
Or create a pop-up to tell that this link may not be safe with "follow the link" or "cancel"

**Comment by bhousel** [comment]
> Or create a pop-up to tell that this link may not be safe with "follow the link" or "cancel"

I don't think that a popup is helpful or friendly.  I'd prefer for iD to be a safe place all the time for people who are maybe not very computer savvy.

Users who really want to visit that link can copy-paste it to a new browser tab and decide if it is worth visiting.  (Consider, the `website` tag value might even contain something like `javascript:alert('hi');` or whatever terrible thing you can imagine).

**Comment by boothym** [comment]
The reason I suggested it was to avoid the need to copy and paste when you want to have a look at a website when editing. I saw the functionality of the wikipedia link and though that would be nice to have. 

I was going to say that iD already gives a red outline for an error when the website is no valid, ie there is no http(s)://. However I noticed that this is done by Firefox due to the type=url input field.

Perhaps iD could do a similar thing and only display the button when the website is a valid url? It would also display it in the bottom left of the browser. 

I do understand the concerns, though according to taginfo, none of the current website values begin with javascript.
